src/components/screens/login-screen.jsx

```diff
@@ -1,8 +1,6 @@
 import React, { useState } from 'react';
 import type { Node } from 'react';
-import {
-    Button, Form, Grid, Image, Message,
-} from 'semantic-ui-react';
+import {Button, Form, Grid, Image, List, Message,} from 'semantic-ui-react';
```

Comment:
Codacy has a fix for the issue: [A space is required after ','.](https://app.codacy.com/gh/armors-dotnet/armors_control_panel/pullRequest?prid=8146859)
```suggestion
import {Button, Form, Grid, Image, List, Message, } from 'semantic-ui-react';
```